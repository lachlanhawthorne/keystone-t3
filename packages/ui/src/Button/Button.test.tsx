import { Button } from "./Button";
import { render, screen } from "@testing-library/react";
import { assert, test, describe, expect, it } from "vitest";
import renderer from "react-test-renderer";

const testRenderer = renderer.create(
  <Button>hello</Button>
);
// @vitest-environment happy-dom
describe("<Button/>", () => {
  it("the title is visible", () => {
    expect(testRenderer.toJSON()).toMatchSnapshot();
    render(<Button>hello</Button>);
    expect(screen.getByText(/hello/i)).toBeTruthy();
  });
});
