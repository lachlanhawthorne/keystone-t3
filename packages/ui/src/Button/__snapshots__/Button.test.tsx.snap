// Vitest Snapshot v1

exports[`<Button/> > the title is visible 1`] = `
<button
  className="c-cuAKdo"
>
  hello
</button>
`;
